src/View/Helper/PaginatorHelper.php

```diff
@@ -1133,6 +1133,11 @@ public function implementedEvents()
      */
     public function limitSelect($limits = [], $default = null, $options = [])
     {
+        if (!in_array('Form', $this->helpers)) {
```

Comment:
When would this case occur? You have included `Form` in `$helpers` property declaration.